.github/workflows/next-build.yml

```diff
@@ -10,9 +10,10 @@
 name: Build & Publish `next`
 
 on:
+  workflow_dispatch:
```

Comment:
Is there a need to run the workflow manually?

Reply:
it may be useful for restart the job if necessary (when it previously failed, or for debug purposes)

Reply:
Agree.